cleanup-db-python.py

```
@@ -0,0 +1,134 @@
+-- Verify deletion
+SELECT COUNT(*) AS remaining FROM GITREPOSUSERSRECORD;
+"""
+
+# Write SQL to temp file
+sql_file = "/tmp/cleanup.sql"
+with open(sql_file, "w") as f:
+    f.write(sql_commands)
+
+# Execute SQL using H2 RunScript tool
+try:
+    result = subprocess.run([
```

Ruff (0.14.14)

[error] `subprocess` call: check for execution of untrusted input

(S603)